translations/it.yml

```diff
@@ -209,6 +209,6 @@ errors:
     phone: '{description} deve essere un numero di telefono valido.'
     url: '{description} deve essere un URL valido '
     time: '{description} deve essere un orario valido (ad es. 10:45)'
-    timeNotPartially: 'Partially times are not supported'
+    timeNotPartially: 'I tempi parziali non sono sopportati.'
```

Comment:
supportati

Sorry, I don't get the meaning of "Partially times", so I don't know what I'm supposed to translate. What are we talking about here?

Reply:
Some browsers allow the user to enter a time only partially. E.g. the user enters the hour but not the minute. This validation error is shown in that case.

Reply:
```suggestion
    timeNotPartially: "L'orario deve specificare sia le ore che i minuti."
```